public/locales/uk/common.json

```diff
@@ -187,6 +187,16 @@
       "cs": "Прочитайте та прослухайте чеські та українські казки",
       "sk": "Прочитайте та прослухайте словацькі та українські казки",
       "pl": "Прочитайте та прослухайте польські та українські казки"
+    },
+    "kids_page_memorygame_title": {
+      "cs": "Грайте в Меморі, озвучену українськими та чеськими словами",
+      "sk": "Грайте в Меморі, озвучену українськими та чеськими словами",
```

Comment:
Tady správně potřebujeme mít v žádku SK a PL místo "českymi" slovenskymi a polskymi, zjistím.

Reply:
FIXED in next commit.

Reply:
To mě nějak vůbec netrklo, díky za opravu.